eva/catalog/models/table_catalog.py

```diff
@@ -20,16 +20,16 @@
 from eva.catalog.models.base_model import BaseModel
 
 
-class DataFrameMetadata(BaseModel):
-    __tablename__ = "df_metadata"
+class TableCatalog(BaseModel):
```

Comment:
Just a curious comment about model in general. Why do we add a catalog suffix for all models? 

To me, catalog includes everything (udf, index, etc). Would this naming give a wrong impression that we maintain different catalogs for different models? (Again, just curious, don’t need a fix).

Reply:
I referred Postgres. They also call each model as different catalog. https://www.postgresql.org/docs/current/catalog-pg-class.html.
What do you think? Maybe I'm wrong.

Reply:
I see. It seems like they use something like `pg_{...}` as their naming convention? I didn't really see they use `catalog` as suffix.